ArcadiaTeamsBot.CQRS/Queries/GetServiceDeskRequestTypesQuery.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+using ServiceDesk.Abstractions.DTOs;
+using System.Collections.Generic;
+
+namespace ArcadiaTeamsBot.CQRS.Queries
+{
+    public class GetServiceDeskRequestTypesQuery : IRequest<IEnumerable<ServiceDeskRequestTypeDTO>>
```

Comment:
Should be in `CQRS.Abstractions` project. Separate abstractions and implementations.

Reply:
Created separate project with abstractions